taui/src/components/route-segments.js

```diff
@@ -15,37 +15,31 @@ export default function RouteSegments ({hasVehicle, routeSegments, travelTime})
   const [bestJourney, ...alternateJourneys] = routeSegments
 
   return (
-    <tbody>
-      <tr className='BestTrip'>
-        <td><span className='fa fa-street-view' /></td>
-        <td>
-          <span>Take </span>
-          {bestJourney.map((segment, index) => (
-            <Segment key={index} segment={segment} />
-          ))}
-          {travelTime > 120
-            ? <span className='decrease'>inaccessible within 120 minutes</span>
-            : <span>in
-              <strong> {travelTime}</strong> {message('Units.Mins')}
-            </span>}
-        </td>
-      </tr>
+    <div className='route-segments'>
+      <div className='route-segments__best-trip'>
+        Take&nbsp;
+        {bestJourney.map((segment, index) => (
+          <Segment key={index} segment={segment} />
+        ))}
+        {travelTime > 120
+          ? <span className='decrease'>inaccessible within 120 minutes</span>
```

Comment:
I noticed this was in `taui`, too; we should move this user-facing string to `messages.yml` (also "Take" above here).